deployment/sriov-network-operator-chart/templates/predeletehook.yaml

```diff
@@ -0,0 +1,24 @@
+{{ if .Values.operator.admissionControllers.enabled }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: "{{ include "sriov-network-operator.fullname" . }}-post-delete-hook"
+  namespace: {{ .Release.Namespace }}
+  annotations:
+    "helm.sh/hook": pre-delete
+    "helm.sh/hook-delete-policy": hook-succeeded,hook-failed
+spec:
+  template:
+    spec:
+      serviceAccountName: {{ include "sriov-network-operator.fullname" . }}
+      containers:
+        - name: cleanup
+          image: bitnami/kubectl:latest
```

Comment:
overall LGTM. i wonder if we should go with that or implement small "cleanup" binary which will do these operations using client-go lib.  having this binary part of the sriov-network-operator image.

@ykulazhenkov @SchSeba @zeeke @Eoghan1232  thoughts ?

Reply:
as a quick workaround, I think approach is probably fine.
but if there is a need to clean up additional resources through something like this, then a binary called from client-go lib would be a better approach probably so that way operator has control.....

Reply:
two points here:
1. yes we need a binary  inside our image. use-case disconnected environments where the user needs to clone images and we have variables in helm to allow him specify the images for the operator. this one is static.
2. add a step in the run CI for k8s to test this